osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -169,11 +174,13 @@ private void updateTooltipText(T value)
             else
             {
                 double floatValue = value.ToDouble(NumberFormatInfo.InvariantInfo);
-                double floatMinValue = CurrentNumber.MinValue.ToDouble(NumberFormatInfo.InvariantInfo);
                 double floatMaxValue = CurrentNumber.MaxValue.ToDouble(NumberFormatInfo.InvariantInfo);
 
-                if (floatMaxValue == 1 && floatMinValue >= -1)
-                    TooltipText = floatValue.ToString("P0");
+                if (DisplayAsPercentage)
+                {
+                    double percentage = floatValue / floatMaxValue;
```

Comment:
Dicey for negative `floatValue`, and kind of nonsensical for negative `floatMaxValue` (even if mathematically correct). Maybe should throw something in those cases?

Reply:
I'm not sure when there would be a use case where the value and max value will both be negative but I think it's still safe enough that it won't warrant an exception. If there is, maybe it would be enough to make another PR of it.